gtest/src/program.rs

```diff
@@ -273,7 +273,10 @@ impl<'a> Program<'a> {
 
         let opt_code = if !is_opt {
             let mut optimizer = Optimizer::new(path).expect("Failed to create optimizer");
-            optimizer.insert_stack_and_export();
+
+            // Ignore result, because it's not important.
+            let _ = optimizer.insert_stack_end_export();
```

Comment:
Underscore can be without `let` but up to you.
```suggestion
            _ = optimizer.insert_stack_end_export();
```

Reply:
Done